packages/arb-token-bridge-ui/tests/e2e/specs/withdraw.cy.ts

```diff
@@ -0,0 +1,173 @@
+import { JsonRpcProvider } from '@ethersproject/providers'
+import { BigNumber } from 'ethers'
+import { formatBigNumber } from '../../../src/util/NumberUtils'
+
+async function getInitialETHBalance(rpcURL: string): Promise<BigNumber> {
+  const goerliProvider = new JsonRpcProvider(rpcURL)
+  return await goerliProvider.getBalance(Cypress.env('ADDRESS'))
+}
+
+const goerliRPC = `https://goerli.infura.io/v3/${Cypress.env('INFURA_KEY')}`
+const arbitrumGoerliRPC = 'https://goerli-rollup.arbitrum.io/rpc'
+
+/**
+ * When user wants to bridge ETH from L1 to L2
+ */
+describe('Withdraw ETH', () => {
+  // Happy Path
+  context('user has some ETH and is on L2', () => {
+    let l1ETHbal
+    let l2ETHbal
+
+    before(() => {
+      cy.clearLocalStorageSnapshot()
+      getInitialETHBalance(goerliRPC).then(
+        val => (l1ETHbal = formatBigNumber(val, 18, 5))
+      )
+      getInitialETHBalance(arbitrumGoerliRPC).then(
+        val => (l2ETHbal = formatBigNumber(val, 18, 5))
+      )
+      cy.disconnectMetamaskWalletFromAllDapps()
+      cy.login()
+    })
+
+    beforeEach(() => {
+      // restore local storage from first test
+      cy.restoreLocalStorage()
+    })
+
+    afterEach(() => {
+      // cypress clears local storage between tests
+      // so in order to preserve local storage on the page between tests
+      // we need to use the cypress-localstorage-commands plugin
+      // or else we have to visit the page every test which will be much slower
+      // https://docs.cypress.io/api/commands/clearlocalstorage
+      cy.saveLocalStorage()
+    })
+
+    it('should show L1 and L2 ETH balances correctly', () => {
+      cy.findByText(`Balance: ${l1ETHbal} ETH`).should('be.visible')
+      cy.findByText(`Balance: ${l2ETHbal} ETH`).should('be.visible')
+    })
```

Comment:
Given we have this both in the deposit and the withdraw tests, maybe we should move it to the login test?